### ISSUE TITLE
can't create notifications, add is disabled

### ISSUE THREAD

**Issue by Dayflare** [ddurieux/glpi_notifications#1]
I can't create notifications with this plugin, add is disabled at notification generation. I tried to reinstall the plugin, doesn't helped.

We use GLPI 9.3.3 on CentOS 7.

![grafik](https://user-images.githubusercontent.com/42730126/49519568-95b6ac00-f8a1-11e8-9409-933b5b82b7a7.png)

**Comment by boscorelly** [comment]
What is your login ?
Are you glpi admin ?

**Comment by boscorelly** [comment]
Can you install a 9.3.1 to test it also ?

**Comment by Dayflare** [comment]
problem persits with latest Update 9.4.0+1.0

i'll try soon with GLPI 9.3.1

**Comment by ddurieux** [comment]
That's really not normal.
Can you try with fresh test environement? I think 9.3 and 9.4 will be the same

**Comment by markotmf** [comment]
I have a same problem.
Server Software: Ubuntu 18.04
	Server Version: 10.1.38-MariaDB-0ubuntu0.18.04.
GLPI 9.4.0

**Comment by boscorelly** [comment]
with a fresh install too ?

**Comment by markotmf** [comment]
instalation is one month old, without update.

Plugins list

 
news                 Name: Alerts                         Version: 1.5.1      State: Enabled
barcode              Name: Barcode                        Version: 2.4.0      State: Enabled
costs                Name: Costs                          Version: 1.1.0      State: Enabled
dashboard            Name: Dashboard                      Version: 0.9.5      State: Enabled
datainjection        Name: File injection                 Version: 2.7.0      State: Enabled
formcreator          Name: Form Creator                   Version: 2.8.0      State: Enabled
addressing           Name: IP Adressing                   Version: 2.8.0      State: Enabled
notifications        Name: Notifications                  Version: 9.4+1.0    State: Enabled
genericobject        Name: Objects management             Version: 2.7.0      State: Enabled
order                Name: Orders management              Version: 2.4.0      State: Enabled
taskdrop             Name: TaskDrop                       Version: 1.1.0      State: Enabled
tasklists            Name: Tasks list                     Version: 1.5.0      State: Enabled

**Comment by markotmf** [comment]
![image](https://user-images.githubusercontent.com/48752817/54674149-1a999e80-4afc-11e9-9b62-a4b5233354ed.png)

**Comment by Hibou-lugubre** [comment]
hello

Same Problem

**Comment by ykasdi** [comment]
Hello, 
Same Problem, any update?

**Comment by ddurieux** [comment]
Not understand what is the problem.

Will check the code to see if I can have the same behavior (not the case for the moment)

**Comment by boscorelly** [comment]
did you tried disabling all addons ? on a fresh install without any addon too ?

**Comment by markotmf** [comment]
![image](https://user-images.githubusercontent.com/48752817/54828514-a1917700-4cb4-11e9-82a0-ea6a19e27430.png)

same issue

![image](https://user-images.githubusercontent.com/48752817/54828605-d7366000-4cb4-11e9-9d54-4d7e5322c4ef.png)

**Comment by boscorelly** [comment]
and with addonds uninstalleds ?

**Comment by ddurieux** [comment]
Perhaps I found the problem, if it's confirmed, I will update the right used for this

**Comment by boscorelly** [comment]
i think it's a conflict with a plugin, i don't have the issue. What did you found ?

**Comment by ddurieux** [comment]
The _config_ profil has no write permission, so it can be this problem, but not sure because I haven't the problem :p

**Comment by boscorelly** [comment]
can you give me the link to this profil ? i don't mind it...

**Comment by ddurieux** [comment]
Try change in this line https://github.com/ddurieux/glpi_notifications/blob/master/inc/notification.class.php#L50 `config` by `notification`

**Comment by Hibou-lugubre** [comment]
Thanks. It work

**Comment by ddurieux** [comment]
So it's a really good news, I will release a corrective version in the 3 next days \o/

**Comment by markotmf** [comment]
NICE !!!!!
It's working now.
Thank you!!!!

**Comment by Dayflare** [comment]
the fix works for me too.
thank you!

now our users can finally enjoy this plugin and the new notifications :)

**Comment by bensonzhou** [comment]
same problem ,   can not create ,,, the + button is disabled ,    my  GLPI VERSION is  9.4.0,,

**Comment by ddurieux** [comment]
With the fix https://github.com/ddurieux/glpi_notifications/issues/1#issuecomment-477357217 too ?

**Comment by bensonzhou** [comment]
it's working  now , need  disable then  enable it ,  thanks！

**Comment by ddurieux** [comment]
I will do the new release this week end

**Comment by juveboss** [comment]
Hello,

The plugin don't work with GLPI 9.4.2 and "notification 9.4+1.0".

=> Add is disabled
=> Before, i had install "notification 9.3.1.0" => Error "add is disabled" also

**Comment by Dayflare** [comment]
@juveboss did you try this fix?

> Try change in this line https://github.com/ddurieux/glpi_notifications/blob/master/inc/notification.class.php#L50 `config` by `notification`

**Comment by juveboss** [comment]
@Dayflare,

Yes i check, the line is changed in version "notification 9.4+1.0" but don't work ...

**Comment by boscorelly** [comment]
have you tried without any other addon ?

**Comment by juveboss** [comment]
Yes i try disable other addon... don't work

**Comment by boscorelly** [comment]
disable is not without addon. Disable = not usable.

**Comment by zezao123** [comment]
Not work for me either, last GLPI release + last plugin release, only with this plugin active

**Comment by benjaminEORIS** [comment]
It's the same problem for us. Any idea to fix this ?
Version GLPI : 9.4.2
Only this plugin is activated.

**Comment by graus69** [comment]
He realizado la moficiacion en notification.class.php, he desactivado el resto de plugins, pero sigue sin funcionar. "La adicion esta desactivada"
Mi version de GLPI es la 9.4.5
